### PR TITLE
Add dao 8 solution

### PR DESCRIPTION
**Pull request by AndreiYazadzhi**

### CODE REVIEW

**Review by IwalkInSilence — APPROVED** [review]

File: src/main/java/com/dao/homework/controllers/CreateCarController.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    protected void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        request.getRequestDispatcher("/WEB-INF/views/createCar.jsp").forward(request, response);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        String carModel = String.valueOf(request.getParameter("model"));
+        Long manufacturerId = Long.valueOf(request.getParameter("manufacturerId"));
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/dao/homework/controllers/CreateCarController.java]:
```suggestion
        Long manufacturerId = Long.valueOf(request.getParameter("manufacturer_Id"));
```

File: src/main/java/com/dao/homework/controllers/CreateDriverController.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    protected void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        request.getRequestDispatcher("/WEB-INF/views/createDriver.jsp").forward(request, response);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        String driverName = String.valueOf(request.getParameter("name"));
+        String driverLicense = String.valueOf(request.getParameter("licenseNumber"));
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/dao/homework/controllers/CreateDriverController.java]:
```suggestion
        String driverLicense = request.getParameter("license_number");
```
change it everywhere

File: src/main/webapp/web.xml [hunk 3]
```diff
@@ -4,15 +4,127 @@
...
+
+    <servlet>
+        <servlet-name>createDriver</servlet-name>
+        <servlet-class>
+            com.dao.homework.controllers.CreateDriverController
+        </servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>createDriver</servlet-name>
+        <url-pattern>/driver/add</url-pattern>
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/web.xml]:
```suggestion
        <url-pattern>/drivers/add</url-pattern>
```

File: src/main/webapp/web.xml [hunk 4]
```diff
@@ -4,15 +4,127 @@
...
+        </servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>deleteDriver</servlet-name>
+        <url-pattern>/drivers/delete</url-pattern>
+    </servlet-mapping>
+
 </web-app>
+
+
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/web.xml]:
```suggestion
```

File: src/main/webapp/web.xml [hunk 5]
```diff
@@ -4,15 +4,127 @@
...
+    <servlet>
+        <servlet-name>addDriverToCar</servlet-name>
+        <servlet-class>
+            com.dao.homework.controllers.AddDriverToCarController
+        </servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>addDriverToCar</servlet-name>
+        <url-pattern>/drivers/add</url-pattern>
```
Comment (dmytro-hryhoruk) [inline src/main/webapp/web.xml]:
```suggestion
        <url-pattern>/cars/drivers/add</url-pattern>
```

**Review by dmytro-hryhoruk — CHANGES_REQUESTED** [review]
let's create directory for each jsp entity
car/all.jsp
car/add.jsp
driver/add.jsp 
and so on..

**Review by dmytro-hryhoruk — APPROVED** [review]
for packages name(jsp packages too) user singular form,
for ur - patterns use plural:
views/car/add.jsp
but the url to controller would be /cars/add

**Pull request merged**